graphistry/umap_utils.py

```diff
@@ -589,10 +589,10 @@ def umap(
             index = res._nodes.index
             if res._node is None:
                 logger.debug("-Writing new node name")
+                res._nodes[config.IMPLICIT_NODE_ID] = range(len(res._nodes))
+
```

Comment:
Clever

Will this work if cudf?

Will this break somehow downstream if .edges is bound and user was using string name IDs for src dst?